src/index.html

```diff
@@ -5,10 +5,59 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Dia</title>
     <link rel="stylesheet" href="./styles/main.css">
+    <link href="https://fonts.googleapis.com/css?family=Poppins:400,600,700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600&display=swap" rel="stylesheet">
   </head>
   <body>
-    <img src="./images/logo.svg" alt="Logo">
-    <h1>Dia</h1>
+    <header class="header">
+      <div class="container">
+        <div class="menu">
+          <nav class="nav menu__nav">
+            <a class="nav__logo" href="#">
+              <img src="./images/logo/dia-logo2.svg" alt="icon-dia">
+            </a>
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a class="nav__link" href="#">Home</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#">About us</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#">Services</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#">Work</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#">Contact us</a>
+              </li>
+            </ul>
+          </nav>
+          <a class="menu__link" href="#">Hire us</a>
+        </div>
+
+        <div class="title">
+          <h1 class="title__head">Strategic<br> Agency</h1>
+          <p class="title__text">By the same illusion which lifts the horizon of the sea to the level of the spectator on a hillside, the sable cloud beneath.</p>
+          <button class="button title__button">Learn more</button>
+        </div>
+      </div>
+
+      <div class="photo-slider">
+        <div class="slider">
```

Comment:
думаю так буде краще, так як це не сам слайдер, а його навігаційні елементи
```suggestion
        <div class="slider-nav">
```

Reply:
Сменил на  slider-nav

Reply:
изменил